src/reporting/clients/ReportsClient.ts

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { AccessToken } from "@itwin/core-bentley";
+import { RequiredError } from "../interfaces/Errors";
+import { EntityListIterator } from "../iterators/EntityListIterator";
+import { EntityListIteratorImpl } from "../iterators/EntityListIteratorImpl";
+import { Collection, getEntityCollectionPage } from "../iterators/IteratorUtil";
+import { OperationsBase } from "../OperationsBase";
+import { Report, ReportCollection, ReportSingle, ReportCreate, ReportUpdate, ReportMapping, ReportMappingCollection, ReportMappingCreate, ReportMappingSingle } from "../interfaces/Reports";
+import { IReportsClient } from "./IReportsClient";
+
+export class ReportsClient extends OperationsBase implements IReportsClient{
+  public async getReports(accessToken: AccessToken, projectId: string, top?: number): Promise<Report[]> {
```

Comment:
Add additional argument `deleted?: boolean = false`. I think it would be best to default to non-deleted reports.

Reply:
added argument deleted: boolean = false (initialized parameters are optional by nature)